src/panels/dev-info/ha-panel-dev-info.js

```diff
@@ -220,6 +221,18 @@ class HaPanelDevInfo extends PolymerElement {
         </template>
       </paper-dialog-scrollable>
     </paper-dialog>
+
+    <paper-dialog with-backdrop id="showcomponents">
```

Comment:
This is not how we do dialogs anymore. Instead, use this https://github.com/home-assistant/home-assistant-polymer/blob/master/src/panels/config/config-entries/ha-config-entries.js#L128-L133

Reply:
And if you want, feel free to put a PR for the dev docs too as it's clearly missing a piece about dialogs.